docs/admin/scim/oauth-protection.md

```diff
@@ -2,16 +2,24 @@
 tags:
   - administration
   - scim
+  - oauth-protection
 ---
 
-## This content is in progress
-
-The Janssen Project documentation is currently in development. Topic pages are being created in order of broadest relevance, and this page is coming in the near future.
-
+## oauth-protection
+  -Launch jans-tui
+  
+  -Navigate to SCIM 
+  
+  -Locate the Scim properties section
+  
+  -Select OAUTH in the Protection Mode list
+  
+  -Click the "Save Configuration" button at the bottom
+  
 ## Have questions in the meantime?
 
 While this documentation is in progress, you can ask questions through [GitHub Discussions](https://github.com/JanssenProject/jans/discussion) or the [community chat on Gitter](https://gitter.im/JanssenProject/Lobby). Any questions you have will help determine what information our documentation should cover.
```

Comment:
You can remove this section too as you have added content now.

Reply:
removed